### PR TITLE
Add Holy Paladin to WowSpec

### PR DESCRIPTION
**Pull request by scamille**
Required for Simc CI builds.

### CODE REVIEW

File: simc_support/game_data/WowSpec.py [hunk 1]
```diff
@@ -392,6 +392,17 @@ def to_dict(self) -> dict:
...
     simc_name="retribution",
 )
+HOLY_PALADIN = WowSpec(
+    id=65,
+    wow_class=WowClass.PALADIN,
+    translations=Language.EmptyTranslation(),
+    talents_blueprint="1100111",
+    raid_role=RaidRole.HEAL,
+    role=Role.MELEE,
+    stat=Stat.STRENGTH,
```
Comment (Bloodmallet) [inline simc_support/game_data/WowSpec.py:402]: `    stat=Stat.INTELLECT,`

**Review by Bloodmallet — CHANGES_REQUESTED** [review]
Mistweaver Monk is missing to fulfill https://github.com/Bloodmallet/simc_support/issues/9

**Comment by Bloodmallet** [comment]
Oh, I just noticed that your PR is against `master`. It should be against `develop`

**Comment by scamille** [comment]
Is develop the default branch for this repository, or is it truly only for development?

If it is the former, consider changing the default branch of the repository.

**Comment by Bloodmallet** [comment]
You're right, I need to clean up the repo and get master updated. My intention was that master reflects the latest release. But because I haven't automated releases for this repo, I was lazy.

**Pull request closed without merging**